addons/stock/controllers/main.py

```diff
@@ -28,10 +29,9 @@ def report(self, output_format, report_name, token, report_id=False, **kw):
                 response.set_cookie('fileToken', token)
                 return response
         except Exception as e:
-            se = _serialize_exception(e)
             error = {
-                'code': 200,
```

Comment:
Despite its value, this code has nothing to do with the HTTP Status Code 200 - OK. Please leave it to 200.